pkg/resources/fluentd/configsecret.go

```diff
@@ -30,9 +30,10 @@ type fluentdConfig struct {
 	LogFormat string
 	LogLevel  string
 	Monitor   struct {
-		Enabled bool
-		Port    int32
-		Path    string
+		Enabled     bool
+		EnabledIPv6 bool
```

Comment:
have you considered using enabledIPv6 by default if it is configured for fluentbit?